cli/src/template.rs

```diff
@@ -184,6 +184,33 @@ describe('{}', () => {{
     )
 }
 
+pub fn package_json() -> &'static str {
+    r#"{
+    "dependencies": {
+        "@project-serum/anchor": "^0.13.2",
+        "chai": "^4.3.4",
```

Comment:
Should both chai and mocha be devDependencies?

Reply:
I suppose they are all technically dev dependencies as they are not required for runtime and only development. Either way I do not mind putting it as dev dependencies. Let me know and I'll get it done.

Reply:
Agreed it doesn't matter much but let's throw them into dev deps.

Reply:
Done